components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -13,6 +13,7 @@
 #include "base/metrics/histogram_macros.h"
 #include "base/strings/string_number_conversions.h"
 #include "base/strings/utf_string_conversions.h"
+#include "base/task/post_task.h"
```

Comment:
still needed?

Reply:
No. O left it accidentally for `sync_thread_->task_runner()->PostTask()` . But this is another thing. Will remove.

Reply:
Thanks. Fixed at https://github.com/brave/brave-core/compare/209f9ce910e51ab9657eafb70c6474ffabe91628..00ba1bd7988f7c5d496b6eed3575aadd362efaee